lib/crowbar/client/config.rb

```diff
@@ -162,11 +162,9 @@ def default_anonymous
       #
       def default_apiversion
         if ENV["CROWBAR_APIVERSION"].present?
-          [
-            1.0, 2.0
-          ].include? ENV["CROWBAR_APIVERSION"]
+          ENV["CROWBAR_APIVERSION"].to_f
```

Comment:
I know this has not been introduced here, but in general it's not a good idea to treat version number as floating point. Is there any good reason do to that here? (And elsewhere in the code)